rdkPlugins/DeviceMapper/README.md

```diff
@@ -0,0 +1,29 @@
+# Dobby RDK Device Mapper Plugin
+
+## Quick Start
+Add the following section to your OCI runtime configuration `config.json` file to enable logging to a file. This will capture all the stdout/err from the container (in addition to any output from `DobbyPluginLauncher`) and redirect it to the specified file.
```

Comment:
This is copy pasted from Logging plugin. Please create proper introduction to Device Mapper plugin.